toolkit/tools/boilerplate/hello/hello.go

```diff
@@ -1,9 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-package hello
```

Comment:
Since we're removing this piece of code as well (and for good reasons, I believe), the commit message seems to vague. There's no indication this got removed.